Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Web/Scripts/blades/export/exporter-list.js

```diff
@@ -13,29 +13,15 @@ angular.module('virtoCommerce.catalogModule')
         var newBlade = {};
         angular.copy(data, newBlade);
 
-        if (data.isGenericExport) {
-            var exportDataRequest = {
-                exportTypeName: 'VirtoCommerce.CatalogModule.Data.ExportImport.ExportableProduct',
-                isTabularExportSupported: true,
-                dataQuery: {
-                    exportTypeName: 'ProductExportDataQuery',
-                    categoryIds : _.pluck(blade.selectedCategories, 'id'),
-                    objectIds: _.pluck(blade.selectedProducts, 'id'),
-                    catalogIds: [blade.catalog.id],
-                    isAllSelected :true
-                }
-            };
-            newBlade.exportDataRequest = exportDataRequest;
-            newBlade.totalItemsCount = (newBlade.selectedProducts || []).length;
-
+        if (angular.isFunction(data.initializeBlade)) {
```

Comment:
Let's do this after 
 newBlade.selectedCategories = …
Also please make following signature for callback:
```
onInitialized(newBlade, { 
 catalog: blade.catalog, 
 selectedCategories: blade.selectedCategories,
 selectedProducts: blade.selectedProducts
})
```